Crashlytics/Crashlytics/Helpers/FIRCLSFeatures.h

```diff
@@ -23,6 +23,7 @@
 #define CLS_USE_SIGALTSTACK (!TARGET_OS_WATCH && !TARGET_OS_TV)
 #define CLS_CAN_SUSPEND_THREADS !TARGET_OS_WATCH
 #define CLS_MACH_EXCEPTION_SUPPORTED (!TARGET_OS_WATCH && !TARGET_OS_TV)
+#define CLS_SIGNAL_SUPPORTED !TARGET_OS_WATCH
```

Comment:
Can you just add a comment saying "As of WatchOS 6 Signal crashes are not supported" and link to the original issue: https://github.com/firebase/firebase-ios-sdk/issues/6434

Reply:
@samedson  Done!